packages/dokan.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,46 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Dokan'
+$category = 'Utilities'
+
+# Bypass linter
+$category
```

Comment:
Please add this package to the linter exclusions instead (the linter should help us writting better code and not making our code uglier 😉 ): https://github.com/mandiant/VM-Packages/blob/main/scripts/test/lint.py#L308

Sorry, I missed it the first time I reviewed it 🙈